AFQ/api.py

```diff
@@ -397,7 +398,8 @@ def _recobundles(row, wm_labels, bundle_dict, reg_template, odf_model="DTI",
                                         force_recompute=force_recompute)
         tg = nib.streamlines.load(streamlines_file).tractogram
         sl = tg.apply_affine(np.linalg.inv(row['dwi_affine'])).streamlines
-        bundles = seg.recobundles(sl, bundle_dict)
+        segmentation = seg.Segmentation(method='Reco')
```

Comment:
We should refactor the API here so that it uses the Segment API directly. That is, the bundles file should call out to segmentation.segment. 

For this, we need to first make sure that the API for the two segmentation functions (`'reco'` and `'ROI'`) are identical, which they are not currently. 

The idea is that the object knows which function it is going to use, but the calling process is agnostic to that, so that the same API can be reused, and we only need one function at this level.